client/src/components/CommentForm/CommentForm.js

```diff
@@ -0,0 +1,256 @@
+import React, {Component} from 'react';
+
```

Comment:
When I edit comment and focus on input, I see inner form to add answer to comment, what is not OK, it would be great to somehow disable that behaviour (maybe stoippropagation when focus, or define on click which is target. If input, do nothing) 
https://i.ibb.co/1KWwjQ6/image.png

Reply:
ok iIwill try